client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/blueprint-form/index.tsx

```diff
@@ -0,0 +1,180 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import Form from 'calypso/a8c-for-agencies/components/form';
+import FormField from 'calypso/a8c-for-agencies/components/form/field';
+import FormRadio from 'calypso/components/forms/form-radio';
+import FormTextarea from 'calypso/components/forms/form-textarea';
+import './style.scss';
+
+type Props = {
+	onContinue: () => void;
+};
+
+type FormData = {
+	topGoal: string;
+	mainGoal2025: string;
+	workModel: string;
+	specificApproach: string;
+};
+
+const BlueprintForm: React.FC< Props > = ( { onContinue } ) => {
+	const translate = useTranslate();
+	const [ formData, setFormData ] = useState< FormData >( {
+		topGoal: '',
+		mainGoal2025: '',
+		workModel: '',
+		specificApproach: '',
+	} );
+
+	const handleSubmit = ( e: React.FormEvent ) => {
+		e.preventDefault();
+		onContinue();
+	};
+
+	return (
+		<Form
+			className="blueprint-form"
+			title={ translate( "Let's build your blurprint" ) }
+			description={ translate(
+				"We'll send you a custom blueprint to grow your business based on your answers below."
+			) }
+		>
+			<div className="blueprint-form__fields">
+				<FormField
+					label={ translate( 'What is your top goal when partnering with a technology provider?' ) }
+					isRequired
+				>
+					<div className="blueprint-form__radio-group">
+						<FormRadio
+							label={ translate( 'Access to cutting-edge technology' ) }
+							checked={ formData.topGoal === 'cutting_edge_tech' }
+							onChange={ () => setFormData( { ...formData, topGoal: 'cutting_edge_tech' } ) }
+						/>
```

Comment:
Can we add `htmlFor` and `id` to all the fields to make the text clickable to check/uncheck the radio.

```suggestion
						<FormRadio
							htmlFor="cutting_edge_tech"
							id="cutting_edge_tech"
							label={ translate( 'Access to cutting-edge technology' ) }
							checked={ formData.topGoal === 'cutting_edge_tech' }
							onChange={ () => setFormData( { ...formData, topGoal: 'cutting_edge_tech' } ) }
						/>
```